src/Components/Youtube/Youtube.tsx

```diff
@@ -20,25 +19,6 @@ function Youtube(	{videoId}: YoutubeProps) {
       ></iframe>
     </div>
   );
-}
-
-
-// const Youtube: React.FC<YoutubeProps> = ({ videoId }) => {
-//   // Construct the URL of the embedded YouTube video
-//   const youtubeUrl = `https://www.youtube.com/embed/${videoId}`;
-
-//   return (
-//     <div className="youtube-card">
-//       <iframe
-//         width="560"
-//         height="315"
-//         src={youtubeUrl}
-//         title="YouTube video player"
-//         allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
-//         allowFullScreen
-//       ></iframe>
-//     </div>
-//   );
-// };
```

Comment:
🔥

Reply:
Hopefully this means something good? 😅

Reply:
Yes haha very good. I am always a fan of cleaning out commented/unused code. Lots of long term benefits to future development to keep your codebase clean.